apps/studio/components/layouts/TableEditorLayout/TableEditorMenu.tsx

```diff
@@ -168,7 +168,7 @@ const TableEditorMenu = () => {
             )}
           </div>
         </div>
-        <div className="flex flex-auto flex-col gap-2 pb-4 px-2">
+        <div className="flex flex-auto flex-col gap-2 px-2">
```

Comment:
Any reason you've removed `pb-4` here? Now the last table is very close to the bottom of the browser.

Reply:
The padding on the bottom seemed off to me with the scroll as well. If you like, I can add it back.

Reply:
What do you mean by "seemed off"? Adding it back would be great or if you can think of a better way that won't seem off that would be great 🙏

Reply:
Just the fact that the scrollbar didn't extend to the bottom of the container. It appeared that the only other way to fix that would have been to add that padding in the InfiniteList component, which would have affected more places in the app.

Reply:
I just added that padding back per your request. Thanks